docs/en/sql-reference/sql-statements/Administration/SET.md

```diff
@@ -24,21 +24,21 @@ SET [ GLOBAL | SESSION ] <variable_name> = <value> [, <variable_name> = <value>]
 
 ## Examples
 
-**Example 1:** Set the `time_zone` to `Asia/Shanghai` within the session.
+Example 1: Set the `time_zone` to `Asia/Shanghai` within the session.
 
 ```Plain
 mysql> SET time_zone = "Asia/Shanghai";
 Query OK, 0 rows affected (0.00 sec)
 ```
 
-**Example 2:** Set the `exec_mem_limit` to `2147483648` globally.
+Example 2: Set the `exec_mem_limit` to `2147483648` globally.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/2e898da0-8c2e-4591-8a0c-20a40047a6ac)
"the xxx parameter", or remove "the" and write only the parameter name "xxx".